source/frontend/src/features/mapSideBar/acquisition/tabs/fileDetails/update/UpdateAcquisitionForm.tsx

```diff
@@ -120,6 +125,27 @@ const AcquisitionDetailSubForm: React.FC<{
     }
   }, [fileStatusTypeCode, setFieldValue]);
 
+  const { organization } = useOrganizationDetail(ownerSolicitorContact?.organizationId);
```

Comment:
@buddy326 
Ok, couple of things here. It looks like with the current design of the contact selector this may be redundant as the contact selector search returns organizations with a nested list of organization persons.

For example, this is the value of ownerSolicitorContact after I select an organization using the contact picker:

{
    "id": "O3",
    "organizationId": 3,
    "organization": {
        "id": 3,
        "isDisabled": false,
        "name": "Dairy Queen Forever! Property Management",
        "organizationPersons": [
            {
                "personId": 3,
                "organizationId": 3,
                "isDisabled": false,
                "rowVersion": 1
            }
        ],
        "organizationAddresses": [],
        "contactMethods": [],
        "rowVersion": 1
    },
    "rowVersion": 0,
    "summary": "Dairy Queen Forever! Property Management",
    "organizationName": "Dairy Queen Forever! Property Management",
    "isDisabled": false
}

As a result, I think you can remove this query entirely, which should also resolve an issue where an error toast would display when clearing the owner solicitor.

Reply:
If I make this change, then the organization persons are each of type Api_OrganizationPerson which does not have the persons details such as name so that would require me to make a call to usePersonDetail() hook for each personId which would be tricky \as react does not like calls to hooks within conditionals like loops so I figured getting the organization details along with all the persons details with just one call is simpler to do.

Reply:
If that is the case you'll need to change your use of useOrganizationDetail.
1) that hook does not get the "concept" form of the model. Going forward we only use models of the "concept" style. Only legacy code is allowed to use non-concept models.
2) that hook also displays a toast error when the org is cleared which doesn't work for this screen.

Reply:
So in this case, what do you mean when you say it does not get the "concept" form of the model? It returns <IEditableOrganization> which contains all the fields we need. I get the second point and agree its not good to have that error pop up to the user's screen but just a little confused about the first point. Also, when you said I'll need to change my use, does that entail changing the implementation of the hook or rather coming up with a workaround in how I call it. Thanks!

Reply:
Yep, so that hook returns a IEditableOrganization which is a legacy code only model. All new code must use the concept models stored in /models/api on the frontend or /concepts on the backend.

In order to do that, you will either need to find an existing hook that returns the organization concept model, or create a new hook that returns the organization concept model.

You should leave the original hook as-is otherwise you would need to rewrite large portions of the contact pages that use hook.

Reply:
ahh I see what you mean, in that case the new hook I create will be returning Api_Organization but another difference between this hypothetical new hook and useOrganizationDetails is that it would not be displaying a toast error to the screen correct? That would resolve the nasty red message from popping up everytime the contact gets cleared. Just want to make sure Im picking up what your putting down haha? Thanks for the thorough replies so far!

Reply:
yes, the red toast is not desirable. Also the new hook (if one is required, please check that one doesn't already exist) should use the new format of using useApiRequestWrapper.

Reply:
ok sweet, I'll get going on it if theres no existing one. Thank you!